mlir-tensorrt/tensorrt/lib/TensorRT/IR/TensorRTVersionCompatibility.cpp

```diff
@@ -185,6 +185,20 @@ bool tensorrt::ElementWiseOp::isValidForTensorRTVersion(
   }
 }
 
+//===----------------------------------------------------------------------===//
+// CastOp
+//===----------------------------------------------------------------------===//
+
+bool tensorrt::CastOp::isValidForTensorRTVersion(int64_t trtMajorVersion) {
+  // TODO (pranavm): Do I need to validate the types here?
```

Comment:
Not sure if any type validation is required here (though `identity` does have it) since we already validate the types in `CastOp::verify`.